pydriller/repository_mining.py

```diff
@@ -133,9 +135,17 @@ def traverse_commits(self) -> Generator[Commit, None, None]:
         for path_repo in self._conf.get('path_to_repos'):
             # if it is a remote repo, clone it first in a temporary folder!
             if self._is_remote(path_repo):
-                tmp_folder = tempfile.TemporaryDirectory()
-                path_repo = self._clone_remote_repos(tmp_folder.name,
-                                                     path_repo)
+                if self._conf.get('clone_repo_to'):
+                    clone_folder = self._conf.get('clone_repo_to')
```

Comment:
I would transform clone_folder to a Path, so Windows doesn't complain. 
```
clone_folder = Path(self._conf.get('clone_repo_to'))

Reply:
Okay I'll look at making this change